src/stars--0.scss

```diff
@@ -0,0 +1,7 @@
+.stars--0 .stars__star {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin: 2px;
+}
```

Comment:
No need to create separate file for each stars block because all of them have identical styles